src/deploy/noobaa_nsfs.service

```diff
@@ -8,7 +8,7 @@ RestartSec=2
 User=root
 Group=root
 ExecStart=/usr/local/noobaa-core/bin/node /usr/local/noobaa-core/src/cmd/nsfs.js
-EnvironmentFile=/etc/noobaa.conf.d/.env
+EnvironmentFile=-/etc/sysconfig/noobaa_nsfs
```

Comment:
This env file is mandatory for system or with `-` it will be optional?

Reply:
yes exactly, -path makes the path optional